src/routes/game/components/elements/gemSlider/GemSlider.tsx

```diff
@@ -21,8 +22,8 @@ const GemSlider = (props: GemSlider) => {
     dispatch(
       submitButton({
         button: {
-          buttonInput: props.cardID,
-          mode: PROCESS_INPUT.TOGGLE_EQUIPMENT_ACTIVE
+          buttonInput: (props.zone != "" ? props.zone + "-" : "") + props.cardID,
+          mode: (props.zone != "" ? PROCESS_INPUT.TOGGLE_PERMANENT_ACTIVE : PROCESS_INPUT.TOGGLE_EQUIPMENT_ACTIVE)
```

Comment:
Is this just to determine what zone the card is in? If it's not in the 'permanents' zone it should act like an equipment? Why the distinction between Auras and Items when parsing the card? Are other permanents then toggled the same as equipment?